src/App.js

```diff
@@ -1,26 +1,48 @@
 import React from 'react';
 import logo from './logo.svg';
 import './App.css';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  NavLink,
+} from 'react-router-dom';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Router>
+        <main>
+          <Switch>
+            <Route path="/listview">
+              <Listview />
+            </Route>
+            <Route path="/additem">
+              <Additem />
+            </Route>
+          </Switch>
+        </main>
+        <footer>
+          <nav>
+            <NavLink to="/listview">
+              <img src="/img/list.svg" alt="view list" />
+            </NavLink>
+            <NavLink to="/additem">
+              <img src="/img/add.svg" alt="add item" />
+            </NavLink>
+          </nav>
+        </footer>
+      </Router>
     </div>
   );
 }
 
+function Listview() {
```

Comment:
I really appreciate that you put these components at the bottom of the file so that `App` is the first component you see when scrolling through. Since it's the most important one here, it's great to focus on it! Nice work 👏